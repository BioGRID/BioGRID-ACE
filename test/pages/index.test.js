
// LIBRARIES
import Vuetify from 'vuetify'

// COMPONENTS
import index from '@/pages/index.vue'

// UTILITIES
import { shallowMount, createLocalVue } from '@vue/test-utils'

const localVue = createLocalVue()

let wrapper

beforeEach(() => {
    const vuetify = new Vuetify()
    wrapper = shallowMount(index, {
        localVue,
        vuetify
    })
})

afterEach(() => {
    wrapper.destroy()
})

describe('pages/index.vue', () => {
    test('renders', () => {
        expect(wrapper.exists()).toBe(true)
    })
    test('renders properly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
