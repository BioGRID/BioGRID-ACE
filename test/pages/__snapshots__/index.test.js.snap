// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index.vue renders properly 1`] = `
"<v-container-stub tag=\\"div\\" fluid=\\"true\\" class=\\"index-page pa-2\\">
  DASHBOARD
</v-container-stub>"
`;
